docs/develop/authentication.md

```diff
@@ -351,11 +361,18 @@ def get_c10l_message(address: str, network_name: Optional[str] = "sapphire-local
     </TabItem>
 </Tabs>
 
-[sp-go]: https://github.com/oasisprotocol/sapphire-paratime/tree/main/clients/go
-[sp-py]: https://github.com/oasisprotocol/sapphire-paratime/tree/main/clients/py
+:::tip
```

Comment:
We should update this line [this line](https://github.com/oasisprotocol/sapphire-paratime/pull/491/files#diff-077b0f866db8adcbb0e034b3e9b30c245375d6348a01d7516b9c3a1a8b121f45L335:~:text=sapphire.wrap(w3)-,w3%20%3D%20sapphire.wrap(w3),-compiled_contract%20%3D%20json). Sapphire middleware is set through .wrap(w3, account) and is separate from construct_sign_and_send_raw_middleware().

Suggested change: `sapphire.wrap(w3, account)`